src/openms/include/OpenMS/ANALYSIS/RNPXL/RNPxlReport.h

```diff
@@ -71,6 +71,7 @@ struct OPENMS_DLLAPI RNPxlReportRow
   double m_3H;
   double m_4H;
   String fragment_annotation_string;  
+  int rank;
```

Comment:
can you sync this here? https://github.com/OpenMS/OpenMS/blob/develop/src/pyOpenMS/pxds/RNPxlReport.pxd#L50

Reply:
can you also add `rank` as a member to the pxd?